Content.Server/Language/Commands/AdminLanguageCommand.cs

```diff
@@ -0,0 +1,75 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.Language;
+using Content.Shared.Language.Components;
+using Content.Shared.Language.Systems;
+using Robust.Shared.Toolshed;
+using Robust.Shared.Toolshed.Syntax;
+using Robust.Shared.Toolshed.TypeParsers;
+
+namespace Content.Server.Language.Commands;
+
+[ToolshedCommand(Name = "language"), AdminCommand(AdminFlags.Admin)]
+public sealed class AdminLanguageCommand : ToolshedCommand
+{
+    private LanguageSystem? _languagesField;
+    private LanguageSystem Languages => _languagesField ??= GetSys<LanguageSystem>();
+
+    [CommandImplementation("add")]
+    public EntityUid AddLanguage(
+        [CommandInvocationContext] IInvocationContext ctx,
+        [PipedArgument] EntityUid input,
+        [CommandArgument] ValueRef<string, Prototype<LanguagePrototype>> @ref,
+        [CommandArgument] bool canSpeak = true,
+        [CommandArgument] bool canUnderstand = true
+        )
+    {
+        var language = @ref.Evaluate(ctx)!;
```

Comment:
What is this?

Reply:
I just did it the same way as it was done in tag command toolshed.